cinema/views.py

```diff
@@ -118,12 +148,18 @@ class OrderPagination(PageNumberPagination):
     max_page_size = 100
 
 
-class OrderViewSet(viewsets.ModelViewSet):
+class OrderViewSet(
+    mixins.ListModelMixin,
+    mixins.CreateModelMixin,
+    viewsets.GenericViewSet
+):
     queryset = Order.objects.prefetch_related(
         "tickets__movie_session__movie", "tickets__movie_session__cinema_hall"
     )
```

Comment:
The `prefetch_related` method is used here to optimize database queries. However, it's not clear why these specific relationships are being prefetched. Please add a comment explaining why these prefetches are necessary.